assets/src/utils/common.ts

```diff
@@ -274,3 +274,25 @@ export const formatNumber = (number: number) => {
 
   return modifiedNumber;
 };
+
+export const isSpecialArrayString = (value: string) => {
```

Comment:
hmm since this "special" behavior is specifically related to capi, I think this method should belong in capi.ts ;